Stratis.Cli/Program.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Linq;
+using NBitcoin;
+using NBitcoin.RPC;
+using Stratis.Bitcoin.Configuration;
+using Stratis.Bitcoin.Features.RPC;
+
+namespace Stratis.Cli
```

Comment:
what do you think of calling this Stratis.Bitcoin.Cli or maybe Stratis.Bitcoin.RPC.Cli?
I find Stratis.Cli a bit too generic.

Reply:
You said this is just a start so I'm not going to review the code here.